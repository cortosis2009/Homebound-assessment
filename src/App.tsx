import React from "react";
import "./App.css";

function App() {
  return (
    <div>
      Insert Team Selector Component
    </div>
  );
};

export default App;
