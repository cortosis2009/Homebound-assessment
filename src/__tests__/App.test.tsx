import React from "react";
import ReactDOM from "react-dom";
import App from "../App";
import { render } from "../test_utils/testRender";


describe("App", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders and matches snapshot", async () => {    
    const c = await render(<App />);
    expect(c.baseElement).toMatchSnapshot();
  });
});