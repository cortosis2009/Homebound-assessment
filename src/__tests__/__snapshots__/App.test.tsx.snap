// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders and matches snapshot 1`] = `
<body>
  <div>
    <div>
      Insert Team Selector Component
    </div>
  </div>
</body>
`;
